site/cypress/e2e/Cypress-Gradeable/rainbow_grading.spec.js

```diff
@@ -0,0 +1,63 @@
+describe('Test Rainbow Grading', () => {
+    beforeEach(() => {
+        cy.login('instructor');
+        cy.visit(['sample', 'config']);
+
+    });
+    it('Enable viewing of rainbow grades and generating the rainbow grading', () => {
+        cy.get('[data-testid="display-rainbow-grades-summary"]').then(($checkbox) => {
+            if (!$checkbox.prop('checked')) {
+                cy.get('[data-testid="display-rainbow-grades-summary"]').check();
+            }
+        });
```

Comment:
Is this all necessary? Are you expecting the state of the VM to be different for each test?